src/quo2/components/buttons/button/properties.cljs

```diff
@@ -3,101 +3,87 @@
 
 (defn custom-color-type
   [customization-color icon-only?]
-  {:icon-color           colors/white
-   :icon-secondary-color colors/white-opa-70
```

Comment:
`icon-secondary-color` is not needed as the icon is the label color for icon only variant 👍